packages/components/form-control/src/index.ts

```diff
@@ -0,0 +1,6 @@
+export * from './FormControl'
+export * from './FormControlState'
+export * from './FormLabel'
+export * from './FormErrorMessage'
+export * from './FormHelperMessage'
+export * from './FormControlContext'
```

Comment:
Shouldn't we harmonize compound declarations, the same way we did on `Tabs` (for eg.):
```tsx
export const FormControl: FC<FormControlProps> & {
  State: FC<FormControlStateProps>
  Label: FC<FormControlLabelProps>
  ErrorMessage: FC<FormControlErrorMessageProps>
  HelperMessage: FC<FormControlHelperMessageProps>
  Context: FC<FormControlContextProps>
} = Object.assign(FormControl, {
  State,
  Label,
  ErrorMessage,
  ...
})
```

Reply:
Yes! I agree. Will do